force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -56,7 +61,8 @@ public with sharing class VariablesDataTypesOperators {
      * @return The product of the two numbers.
      */
     public static Integer multiplication(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        Integer result = a * b;
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Try returning this in one line

Reply:
Updated return statement to consolidate into one line